### PR TITLE
Digital #climatestrike

### PR DESCRIPTION
**Pull request by znarf**
Banner (in french because FR browser)

Capture d’écran 2019-09-04 à 18 28 24

Full Screen

Capture d’écran 2019-09-04 à 16 50 25

### CODE REVIEW

**Comment by xdamman** [comment]
I don't see the banner on the deployed site: https://opencollective-frontend-edrqutzqvu.now.sh/queerjs

**Comment by znarf** [comment]
@xdamman try with the new deployment https://opencollective-frontend-ojkirwfhni.now.sh/queerjs

**Comment by xdamman** [comment]
Great. Can we host ourselves the static HTML page in the iframe instead of loading it from an external server? Also, I'd love to change the update the text. We need to link it to the blog post that we will post for example.

**Comment by znarf** [comment]
@xdamman that increases the complexity

**Comment by xdamman** [comment]
There is an option to specify the host for the iframe: https://github.com/opencollective/opencollective-frontend/pull/2487/files#diff-bee5869738e0f49776a1aae21b9a0bf0R10

**Comment by znarf** [comment]
@xdamman maybe you can try to fork their repository and deploy your fork.

**Comment by xdamman** [comment]
Just spent last hour trying to figure that out but couldn't.

Can you take a look and try to deploy it? 
https://github.com/opencollective/digital-climate-strike

This is what I get:

```
Error: No output directory named "public" found.
--
Sep 04 2019 18:46:55:883 | More details: https://zeit.co/docs/v2/advanced/platform/frequently-asked-questions#missing-public-directory
Sep 04 2019 18:46:55:883 | at validateDistDir (/tmp/453ec098a39e89fa/.build-utils/.builder/node_modules/@now/static-build/dist/index.js:264:15)
```

(and I looked at https://zeit.co/docs/v2/advanced/platform/frequently-asked-questions#missing-public-directory but couldn't figure it out)

File: pages/_app.js [hunk 1]
```diff
@@ -69,11 +69,30 @@ class OpenCollectiveFrontendApp extends App {
     // See https://github.com/formatjs/react-intl/issues/254
     const initialNow = Date.now();
 
-    return { pageProps, scripts, initialNow, locale, messages };
+    const digitalClimateStrikeBannerEnabled = true;
+    const digitalClimateStrikeFullpageEnabled = ctx.req && ctx.req.url === '/';
+    const digitalClimateStrikeOptions = {
+      cookieExpirationDays: 30,
```
Comment (xdamman) [inline pages/_app.js]: we also need to add `websiteName: "Open Collective",` as an option

**Comment by znarf** [comment]
@xdamman it doesn't build locally (`npm run build`), I think it's due to your modification.

For the deployment, the FAQ https://zeit.co/docs/v2/advanced/platform/frequently-asked-questions#missing-public-directory is pretty clear, you just need to make sure the build is creating a `public` directory with your static site in there.

**Comment by xdamman** [comment]
My changes are only text (and sadly some indenting because of the config of my VSCode). So I don’t understand. And I also couldn’t figure out how to configure Webpack to make it build into a `public` directory. I guess I’m getting rusty.

Can you take it from here?

**Comment by znarf** [comment]
It would be great if you stayed engaged in that project! :-)

- You should escape the single quote in "Plus d'info" -> "Plus d&#39;info"
- Why not just add `&& cp dist build` at the end of the `build` in `package.json`?

**Comment by xdamman** [comment]
Believe me I’m very engaged in that project ;-)
Just not on the technical level. We all have different skills to contribute here. It’s just that all my edits in various google docs and spreadsheets and keynote presentations don’t show up as commits on github ¯\_(ツ)_/¯

**Comment by xdamman** [comment]
Ok great it worked! https://digital-climate-strike-xdamman.opencollective.now.sh/
But I guess it has to be loaded with the options. Don't forget to add 
`websiteName: 'Open Collective'`

**Comment by xdamman** [comment]
Alias: https://oc-digital-climate-strike.now.sh

**Pull request merged**